onnxruntime/python/tools/transformers/dev_benchmark.cmd

```diff
@@ -94,17 +101,24 @@ if %run_gpu_fp32% == true (
 if %run_gpu_fp16% == true (
   for %%m in (%models_to_test%) DO (
     echo Run GPU FP16 Benchmark on model %%m
-    call :RunOneTest %%m -g --fp16
+    call :RunOneTest %%m -g -p fp16
```

Comment:
Do we need to add -p fp32 for line 97?

Reply:
Never mind. Just saw fp32 is the default argument

Reply:
-p fp32 is the default so it can be omitted.